### PR TITLE
tutorial on image annotations

### PR DESCRIPTION
**Pull request by emmanuelle**
This PR takes the content (the tutorial on image annotations) from #1004, where the code has been reformatted to fit in the traditional structure of dash-docs chapters. Therefore it should be ready for review.

I'll leave #1004 open for a while to discuss about the possibility to include notebooks as source files for the documentation.

### CODE REVIEW

File: dash_docs/chapter_index.py [hunk 1]
```diff
@@ -897,7 +912,8 @@ def component_list(
                         'content': chapters.dash_canvas.index.layout,
                         'description': (
                             'Image rendering, drawing, annotations '
-                            'for image processing applications.'
+                            'for image processing applications'
```
Comment (emmanuelle) [inline dash_docs/chapter_index.py:915]:
```suggestion
                            'for image processing applications '
```

**Comment by emmanuelle** [comment]
I think the PR is ready for review, with CI passing. (percy changes come from a new item in the sidebar)

If the tutorial is merged before my webinar next Wednesday, I'll add a link to the corresponding page in my webinar slides.

**Comment by emmanuelle** [comment]
Should I go ahead and accept the percy changes? It's all related to the new section (nav bar, table of contents page + of course new page).

**Review by alexcjohnson — APPROVED** [review]
Excellent, thanks! 💃

**Pull request merged**